docs/pdf-lesson-plan-generation.md

```diff
@@ -57,3 +57,10 @@ For Dropbox folder access, ask Brendan or anyone with Dropbox `Pegasus Staging`
   brew install https://raw.github.com/quantiverge/homebrew-binary/pdftk/pdftk.rb
   brew install enscript
   ```
+
+## To run PDF collation tests
+
+1. In `/Gemfile`, un-comment the line with `gem 'pdf-reader'`
+1. `bundle install`
+1. `cd code-dot-org/pegasus/test/disabled`
+1. `bundle exec ruby -Itest test_pdf_merger.rb`
```

Comment:
Hm, I don't recall exactly why (maybe @philbogle or @breville might remember what the issue was? we moved `test_pdf_merger.rb` to `test_pdf_merger.rbdisabled`), I think one of the dependencies was causing issues for some subset of the dev team and the tests would then not pass or throw errors for them.

In this PR I moved the test back to a .rb file but into a non-auto-run folder so it gets lint covered by lint and is editable/runnable.

On reflection it does make sense to try re-enabling in this branch, and then I'll merge next Thursday once I'm back from vacation so can respond if errors pop up for others.